examples/flax_mnist.py

```diff
@@ -15,16 +15,33 @@
 """MNIST example with Flax and JAXopt."""
 
 from absl import app
+from absl import flags
+
 from flax import linen as nn
+
 import jax
 import jax.numpy as jnp
+
 from jaxopt import loss
 from jaxopt import OptaxSolver
+from jaxopt import PolyakSGD
 from jaxopt import tree_util
+
 import optax
+
 import tensorflow_datasets as tfds
 
 
+flags.DEFINE_float("l2reg", 1e-4, "L2 regularization.")
+flags.DEFINE_float("learning_rate", 0.001, "Learning rate (used in adam).")
+flags.DEFINE_bool("manual_loop", False, "Whether to use a manual training loop.")
```

Comment:
very minor and feel free to ignore:  still not convinced with calling it a manual training loop. How about "explicit" instead of manual?

Reply:
I personally have a slight preference for "manual" (in the sense of "done by hand"), the opposite of automatic.